composer.json

```diff
@@ -10,18 +10,20 @@
     ],
     "require": {
         "php": "^5.6 || ^7.0",
-        "php-http/httplug": "^1.0",
+        "php-http/httplug": "^1.0 || ^2.0",
         "php-http/client-implementation": "^1.0",
-        "php-http/discovery": "^1.0",
         "psr/log": "~1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.36 || ^5.5 || ^6.2",
+        "phpunit/phpunit": "^4.8.36 || ^5.5 || ^7.5 || ^8.4",
         "nyholm/nsa": "^1.1",
-        "php-http/curl-client": "^1.7",
-        "php-http/message": "^1.4",
-        "guzzlehttp/psr7": "^1.4"
+        "php-http/curl-client": "^1.0 || ^2.0",
+        "php-http/discovery": "^1.7",
+        "php-http/message": "^1.8",
+        "nyholm/psr7": "^1.2"
     },
+    "minimum-stability": "stable",
+    "prefer-stable" : true,
```

Comment:
What was the reason for adding these `prefer-stable` and `minimum-stability`?

Reply:
Hm.. Im not sure. 
I guess other library have this to make it simpler to get beta versions if no other option are available. 

Im not a big fan of too many unrelated changes in one PR.